src/DigitalSubscriberAppBanner.tsx

```diff
@@ -0,0 +1,332 @@
+import React, { useState } from 'react';
+import { css } from "@emotion/core";
+import { palette, space } from "@guardian/src-foundations";
+import { from, until } from '@guardian/src-foundations/mq';
+import { body, headline } from '@guardian/src-foundations/typography/cjs';
+import { Button } from "@guardian/src-button";
+import { SvgCross, SvgInfo } from '@guardian/src-icons';
+import { AppStore } from "./assets/app-store"
+import { PlayStore } from "./assets/play-store"
+
+const imgHeight = "300";
+
+export type Props = {
+  message: string;
+  onButtonClick: (buttonType: string) => void;
+  firstName?: string;
+};
+
+export const wrapper = css`
+  html {
+    box-sizing: border-box;
+  }
+  *,
+  *:before,
+  *:after {
+    box-sizing: inherit;
+  }
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  background-color: ${palette.background.secondary};
+  color: ${palette.neutral[20]};
+`;
+
+export const contentContainer = css`
+    align-items: stretch;
+    display: flex;
+    flex-direction: column;
+    position: relative;
+    margin: 0 auto;
+    width: 100%;
+    max-width: 980px;
+    ${from.tablet} {
+        flex-direction: row;
+    }
+    ${from.desktop} {
+        min-height: ${imgHeight}px;
+    }
+    ${from.leftCol} {
+        max-width: 1140px;
+    }
+    ${from.wide} {
+        max-width: 1300px;
+    }
+`;
+
+export const topLeftComponent = css`
+    width: 93%;
+    padding: ${space[4]}px;
+    ${from.tablet} {
+        width: 55%;
+        padding-right: 0;
+    }
+    ${from.desktop} {
+        width: 60%;
+    }
+`;
+
+export const bottomRightComponent = css`
+    display: flex;
+    justify-content: center;
+    width: 100%;
+    max-height: 100%;
+
+    ${from.tablet} {
+        align-self: flex-end;
+        max-width: 40%;
+        padding-right: ${space[4]}px;
+        padding-right: ${space[4]}px;
+    }
+
```

Comment:
This is minor to the point of irrelevance, but should we choose a code style for these blocks? Above, the breakpoints are all smooshed together but we have new lines between each breakpoint here.

I'm not fussed in the slightest, just sharing something I've noticed.

Reply:
Yes, definitely worth going for something consistent here. I've reorganised, going for a couple of line breaks before nested blocks.